x-pack/plugins/enterprise_search/common/types/index.ts

```diff
@@ -66,6 +66,9 @@ export interface ClientConfigType {
   host?: string;
   ui: {
     enabled: boolean;
+    queryBuilder: {
```

Comment:
this doesn't actually exist on the enterprise_search config, only the search_playground config correct?

Reply:
Correct. for now.